lib/fog/kubevirt/models/compute/template.rb

```diff
@@ -2,39 +2,184 @@ module Fog
   module Compute
     class Kubevirt
       class Template < Fog::Model
-        identity :name,             :aliases => 'metadata_name'
-
-        attribute :namespace,       :aliases => 'metadata_namespace'
-        attribute :description,     :aliases => 'metadata_description'
-        attribute :tags,            :aliases => 'metadata_tags'
-        attribute :labels,          :aliases => 'metadata_labels'
-        attribute :resourceVersion, :aliases => 'metadata_resourceVersion'
-        attribute :uid,             :aliases => 'metadata_uid'
-        attribute :cpu_cores,       :aliases => 'spec_cpu_cores'
-        attribute :memory,          :aliases => 'spec_memory'
-        attribute :disks,           :aliases => 'spec_disks'
-        attribute :volumes,         :aliases => 'spec_volumes'
+        identity :name,             aliases: 'metadata_name'
+
+        attribute :namespace,       aliases: 'metadata_namespace'
+        attribute :description,     aliases: 'metadata_description'
+        attribute :tags,            aliases: 'metadata_tags'
+        attribute :labels,          aliases: 'metadata_labels'
+        attribute :resourceVersion, aliases: 'metadata_resourceVersion'
```

Comment:
per naming convention, shouldn't it be ``resource_version````

Reply:
Done